src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js

```diff
@@ -38,15 +40,30 @@ export const BulkEditInApp = ({
   const { holdingsNotes, isHoldingsNotesLoading } = useHoldingsNotes({ enabled: isHoldingsRecordType });
 
   const optionsMap = {
-    [CAPABILITIES.ITEM]: getItemsOptions(intl.formatMessage, itemNotes),
-    [CAPABILITIES.USER]: getUserOptions(intl.formatMessage),
-    [CAPABILITIES.HOLDING]: getHoldingsOptions(intl.formatMessage, holdingsNotes),
-    [CAPABILITIES.INSTANCE]: getInstanceOptions(intl.formatMessage),
+    [CAPABILITIES.ITEM]: getItemsOptions(formatMessage, itemNotes),
+    [CAPABILITIES.USER]: getUserOptions(formatMessage),
+    [CAPABILITIES.HOLDING]: getHoldingsOptions(formatMessage, holdingsNotes),
+    [CAPABILITIES.INSTANCE]: getInstanceOptions(formatMessage),
   };
 
   const options = optionsMap[currentRecordType];
   const showContentUpdatesForm = options && !isItemNotesLoading && !isHoldingsNotesLoading;
-  const sortedOptions = sortAlphabetically(options, intl.formatMessage({ id:'ui-bulk-edit.options.placeholder' }));
+  const sortedOptions = sortAlphabetically(options, formatMessage({ id:'ui-bulk-edit.options.placeholder' }));
+
+  const defaultOptionValue = options[0].value;
+
+  const fieldTemplate = {
+    options,
+    option: defaultOptionValue,
+    actionsDetails: getDefaultActions({
+      option: defaultOptionValue,
+      capability: currentRecordType,
+      options,
+      formatMessage
```

Comment:
Do we need to pass `formatMessage` from `intl` here?

Reply:
Yes, there is some logic related to generationg actions, and it includes parts which have to be translated